### DISCUSSION TITLE
Newbie question

### DISCUSSION THREAD

**Discussion by kkorotev** [arkypita/LaserGRBL#1114] [Q&A]
How do I quickly re-size my (target area?) to accommodate the artwork and wood piece sizes I use.
Kevin

**Comment by StuartB4** [comment]
In Target image settings you change the W and H settings to the size you want on the work piece. But if you change one the other will change so it keeps the image in proportion.

Or did you mean the size the engraver can do.

**Comment by kkorotev** [comment]
I think I meant quickly.  The poor UI which came with the unit has a very fast way of re-sizing.  GRBL does not seem to?

**Comment by Jay-F61** [comment]
So if I'm understanding this when I set up Grbl it set the work area as 200mm x200mm.  In the second step if I change the W & H settings it will override that.

**Comment by StuartB4** [comment]
@Jay-F61 

The size of the area the laser can engrave should be set in grbl configuration.  $130 for the X axis and $131 for the Y axis. 
That's the maximum travel of both axis's.

You load your image and prepare it for engraving.
When you get to Target image settings you can set the Width and Height with the W and H settings to
fit on the material you want to engrave.
Just make sure you dont  exceed the size set in $130 and $131 in grbl configuration.

Mine are set for 285mm's for X and 385mm's for Y. So it will engrave upto that size.
If I try to engrave an image bigger than that I get a red box around the numbers in the W or H boxes as a warning that they have exceeded the grbl settings.

Whan you change either W or H the other one will automatically change to keep the aspect ratio of the image, that prevents distortion of the engraved image.

**Comment by Jay-F61** [comment]
Thanks. I will give it a try.
